addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -98,6 +98,34 @@ def test_event_booth_prices_with_sale_order(self):
                 booth.state, 'unavailable',
                 "Booth should not be available anymore.")
 
+    @users('user_sales_salesman')
+    def test_event_booth_count(self):
+        # Create sale order and add a booth
+        sale_order = self.env['sale.order'].create({
```

Comment:
Would test a bit more things here: 
- First adapt base test data to have 3 booths instead of 2
- In the initial sale order, select booth 1 and 2
- Check that the count is 2 AND that we have existing event.booth.registrations for booth 1 and 2
- Then update the sale.order to unselect booth 1 and select booth 3 instead
- Check that the count is still 2 AND that we have existing event.booth.registrations for booth 1 and 3 AND that the one for booth 2 is unlinked

That way we have more coverage.

Reply:
Hello @awa-odoo :slightly_smiling_face: ,
Thanks for the review.
I agree with your points. I have updated the code. Can you please have a look?